contrib/scripts/umeemainnet_fork.sh

```diff
@@ -6,7 +6,7 @@
 # upgrade this fork with a software-upgrade proposal.
 
 # USAGE: ./umeemainnet_fork.sh
-set -e
+set +e
```

Comment:
Testing the effect of this on v1 -> v3 script. Hoping to more reliably reveal failures.